packages/canary-react/src/stories/ic-data-table.stories.js

```diff
@@ -237,6 +238,43 @@ export const DisableSort = {
   name: "Disable sort",
 };
 
+/**
+ * Alternatively, set `disableAutoSort` to `true` within the column object to have it apply only to certain columns that may require special custom sorting.
+ */
+export const DisableAutoSortOnColumns = {
+  render: () => {
+    const ExternalData = [...DATA];
+
+    const handleSort = ({ columnName, sorted }) => {
+      console.log("Sort changed", columnName, sorted);
+      if (columnName !== "firstName") return;
+
+      const sortedAscending = sorted === "ascending";
+      ExternalData.sort((a, b) => {
+        if (a[columnName] < b[columnName]) return sortedAscending ? -1 : 1;
+        if (a[columnName] > b[columnName]) return sortedAscending ? 1 : -1;
+        return 0;
+      });
+      console.log("Custom sort applied");
+    };
+
+    return (
+      <IcDataTable
+        columns={COLS_DISABLE_AUTO_SORT}
+        data={ExternalData}
+        caption="Disable auto sort on columns"
+        sortable
+        disableAutoSort
```

Comment:
is this needed as it's presumably turning it off for all columns & hence overriding the column level setting?